delta/sdk/src/test/scala/ch/epfl/bluebrain/nexus/delta/sdk/directives/AuthDirectivesSpec.scala

```diff
@@ -83,8 +83,12 @@ class AuthDirectivesSpec
       }
     }
 
+  private val authExceptionHandler: ExceptionHandler = ExceptionHandler { case AuthorizationFailed =>
```

Comment:
It would be better to use an ExceptionHandler defined in for the app

Reply:
`ExceptionHandler` is in `app` module and this test is in `sdk` module. Unless we want to move `ExceptionHandler` (and  probably the `RejectionHandler`), to `sdk` ,we can't really  use them here.

Reply:
Yep, you are right